packages/ckan/src/namespace.js

```diff
@@ -0,0 +1,8 @@
+import _rdf from '@zazuko/env'
+import prefixes from '@zazuko/prefixes'
+
+export const dcat = _rdf.namespace(prefixes.dcat)
+export const dcterms = _rdf.namespace(prefixes.dcterms)
+export const rdf = _rdf.namespace(prefixes.rdf)
+export const schema = _rdf.namespace(prefixes.schema)
+export const skos = _rdf.namespace(prefixes.skos)
```

Comment:
Zazuko env has all that. Might even remove the need for dependency on `@zazuko/prefixes`?

```suggestion
import rdf from '@zazuko/env'

const { dcat, dcterms, rdf, schema, skos } = rdf.ns

export { dcat, dcterms, rdf, schema, skos }
```

Reply:
I can't remove the dependency for `@zazuko/prefixes` as it's used in `src/xml.js`:

```js
import prefixes, { shrink } from '@zazuko/prefixes'
```

I also needed to rename `rdf` to `_rdf` back, as it's already in the export.